### ISSUE TITLE
getting the value of & in functions

### ISSUE THREAD

**Issue by geddski** [sass/sass#1368]
How would you get the string value of the parent inside of a function? I tried several variations of this:

``` scss
@function getParentName() {
  @return &;
}
```

The end goal is to pass this string value into the built in string function `str-slice`. Is this possible?

**Comment by ezekg** [comment]
If I understand you correctly, you don't want a list returned, correct?
Interpolating it (`#{}`) will cast it into a string.

``` scss
@function getParentName() {
  @return #{&};
}
```

**Comment by geddski** [comment]
That throws an error in sass 3.3.14:

```
Syntax error: Invalid CSS after "  @return #{": expected expression (e.g. 1px, bold), was "&};"
```

@chriseppstein could we reopen this?

**Comment by ezekg** [comment]
@geddski update Sass to 3.4.x, as it [introduced the `&` back into SassScript](https://github.com/sass/sass/blob/master/doc-src/SASS_CHANGELOG.md?utm_content=buffer8987d&utm_medium=social&utm_source=twitter.com&utm_campaign=buffer#340-unreleased); that feature isn't available in 3.3 for reasons explained [here](https://gist.github.com/nex3/8050187). I have a project that uses a custom Ruby function as a polyfill though. To use it, add this to the bottom of your `config.rb` file.

``` ruby
# Custom functions
module Sass::Script::Functions
    # Returns stringified selector
    # ----
    # @return [string]
    def getParentName()
        Sass::Script::String.new(environment.selector.to_s)
    end
end
```

You can call it the same way you would call a normal Sass function.
